ringpop.go

```diff
@@ -173,11 +174,44 @@ func (rp *Ringpop) init() error {
 	rp.forwarder = forward.NewForwarder(rp, rp.subChannel)
 	rp.forwarder.RegisterListener(rp)
 
+	rp.startTimers()
 	rp.setState(initialized)
 
 	return nil
 }
 
+// Starts periodic timers in a single goroutine. Can be turned back off via
+// stopTimers. At present, only 1 timer exists, to emit ring.checksum-periodic.
+func (rp *Ringpop) startTimers() {
+	if rp.tickers != nil {
+		return
+	}
+	rp.tickers = make(chan *time.Ticker, 1) // 1 == max number of ticker
```

Comment:
[discussed offline] - is it more gopherish to use channel here? I found at least one go blogger claiming as much. Either is possible.